samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,351 @@
+   Sets the stack size (in bytes) for the connection thread of the sample.
+
+.. _CONFIG_MESSAGE_THREAD_STACK_SIZE:
+
+CONFIG_MESSAGE_THREAD_STACK_SIZE - Message Queue Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the message queue processing thread of the sample.
+
+.. _CONFIG_MAX_OUTGOING_MESSAGES:
+
+CONFIG_MAX_OUTGOING_MESSAGES - Outgoing message maximum
+   Sets the maximum number of messages that can be in the outgoing message queue.
+   Messages submitted past this limit will not be enqueued.
+
+.. _CONFIG_MAX_CONSECUTIVE_SEND_FAILURES:
+
+CONFIG_MAX_CONSECUTIVE_SEND_FAILURES - Max outgoing consecutive send failures
+   Sets the maximum number of device messages which may fail to send before a connection reset and cooldown is triggered.
+
+.. _CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS:
+
+CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS - Cooldown after max consecutive send failures exceeded (seconds)
+   If a connection reset is triggered by too many failed device messages, the sample will wait this long (in seconds) before trying again.
+
+.. _CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS:
+
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS - Sensor sampling interval (seconds)
+   Sets the time to wait between each temperature sensor sample.
+
+.. warning::
+   Every time a sensor sample is sent, it will interrupt any attempted GNSS fix.
+   If you set this interval too short, there will not be enough time to ever acquire a fix.
+   In good conditions, and with AGPS data, one minute is generally just barely long enough to get a fix most of the time.
+
+.. _CONFIG_GNSS_ANTENNA_ONBOARD:
+
+CONFIG_GNSS_ANTENNA_ONBOARD - Select the onboard GNSS antenna (default)
+   This option enables the onboard GNSS antenna.
+
+
+.. _CONFIG_GNSS_ANTENNA_EXTERNAL:
+
+CONFIG_GNSS_ANTENNA_EXTERNAL - Select an external GNSS antenna
+   This option enables the external GNSS antenna.
+
+
+.._CONFIG_GNSS_AT_MAGPIO:
+
+CONFIG_GNSS_AT_MAGPIO - ``AT%XMAGPIO`` command
+   Sets the ``AT%XMAGPIO`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
+
+.._CONFIG_GNSS_AT_COEX0:
+
+CONFIG_GNSS_AT_COEX0 - ``AT%XCOEX0`` command
+   Sets the ``AT%XCOEX0`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
+
+.._CONFIG_GNSS_FIX_TIMEOUT_SECONDS:
+
+CONFIG_GNSS_FIX_TIMEOUT_SECONDS - GNSS Fix timeout (seconds)
+   On each location sample, try for this long to achieve a GNSS fix before falling back to cellular positioning.
+
+.. warning::
+	Set this to 20 or so seconds below the sensor sample interval so that there is time left over to perform cellular positioning.
+	Otherwise, location samples may occur at a longer interval than requested.
```

Comment:
```suggestion
.. note::
	Set this to approximately 20 seconds below the sensor sample interval so that there is time left to perform cellular positioning.
	Otherwise, location samples may occur at a longer interval than requested.
```